tests/parsers/google_logging.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for Google log parser"""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.lib import errors
+from plaso.parsers import google_logging
+
+from tests.parsers import test_lib
+
+
+class GooglelogTest(test_lib.ParserTestCase):
+  """Tests for the Google logging parser"""
+
+  def testEventsParsed(self):
```

Comment:
Can we add tests for the individual methods as well. So that we know these are properly tested for error cases as well.

I'm wondering why this parser unittest is deviating so much from other ones? Is this a left over from earlier work?

Reply:
Changed this to be more like the other parser tests.